lib/model.js

```diff
@@ -386,6 +386,22 @@ lastStepModifierFunctions.$inc = function (obj, field, value) {
   }
 };
 
+/**
+ * Updates the value of the field, only if specified field is greater than the current value of the field
+ */
+lastStepModifierFunctions.$max = function (obj, field, value) {
+  if (typeof obj[field] === 'undefined') obj[field] = value;
+  else if (value > obj[field]) obj[field] = value;
```

Comment:
In terms of style nedb always uses curly braces around `if` and `else` body, even when it's only one line. In this case here is how this if should be written (I'll do the rewrite during merge):  

```javascript
if (typeof obj[field] === 'undefined') {
  obj[field] = value;
} else if (value > obj[field]) {
  obj[field] = value;
}
```